tests/test_datetimes.py

```diff
@@ -167,28 +167,13 @@ def test_manual_increment():
     with freeze_time(initial_datetime) as frozen_datetime:
         assert frozen_datetime() == initial_datetime
 
-        frozen_datetime.tick()
-        initial_datetime += datetime.timedelta(seconds=1)
-        assert frozen_datetime() == initial_datetime
-
-        frozen_datetime.tick(delta=datetime.timedelta(seconds=10))
-        initial_datetime += datetime.timedelta(seconds=10)
-        assert frozen_datetime() == initial_datetime
-
-
-def test_manual_increment_seconds():
```

Comment:
Note that `test_manual_increment_seconds` and `test_manual_increment` were doing the same exact thing. So i deleted one of them.